### ISSUE TITLE
MNTP learning rate

### ISSUE THREAD

**Issue by spookyQubit** [McGill-NLP/llm2vec#76]
Great work. Really appreciate that the code is public (and also the paper is so clearly written).

I had a question about the MNTP config. Can you please confirm what learning rate was used when running `run_mntp.py` for llama2/llama3? Looking at the configs under `train_configs`, it seems that the default learning rate in TrainingArguments of 5e-5 was used? 

Sorry if this is already in the paper and I missed it (I read appendix D.1.1 but I am not sure where to look at the details for `same training parameters as RoBERTa MNTP training`).

-- Thanks.

**Comment by vaibhavad** [comment]
Hi @spookyQubit, thanks for your interest in our work.

We modified the [huggingface mlm training script](https://github.com/huggingface/transformers/blob/481a95781404e48b1c80940be17e8279dec82fe8/examples/pytorch/language-modeling/run_mlm.py). Similar to this script, we used the default learning rate in TrainingArguments which is 5e-5.

**Comment by spookyQubit** [comment]
Thanks @vaibhavad .
